### DISCUSSION TITLE
July 17th Office Hours: Nebular, Docs, Timers, high-fidelity integration testing

### DISCUSSION THREAD

**Discussion by Jovonni** [Agoric/agoric-sdk#9723] [Office Hours]
# Links
- https://docs.agoric.com/guides/orchestration/
- https://docs.google.com/presentation/d/1Zh8t5dDQTknc3wMfDm0MGHRyGLgzI_7r4BlG4GicPRM/edit?usp=sharing
- https://agoric-sdk.pages.dev/modules/_agoric_time#md:timerbrand
- https://github.com/Agoric/agoric-sdk/blob/c7a5fd7dc684f4fbaf9b72f6f222d9aa06b695ba/packages/SwingSet/test/timer/bootstrap-timer.js
- https://github.com/Agoric/agoric-sdk/blob/c7a5fd7dc684f4fbaf9b72f6f222d9aa06b695ba/packages/SwingSet/tools/manual-timer.js#L82

**Comment by LuqiPan** [comment]
There's also

https://github.com/Agoric/agoric-sdk/blob/master/packages/zoe/tools/manualTimer.js#L74

which wraps SwingSet's buildManualTimer
